src/ExceptionTransformer.ts

```diff
@@ -46,9 +68,127 @@ class ExceptionTransformer {
         exception.fallback_message
       );
     }
-
     return exceptionMap;
   }
+
+  /*  Returns `getError` function, which returns the field error if there is errorDetail.
+      Otherwise returns () => undefined
+      
+      `getError` function returns value of given key if `fieldName` is string
+      `fieldName` can be "message.title.name" and this function can through
+      errorInfo = {
+        message: {
+          title: {
+            name: ["a logical message"];
+          }
+        }
+      }
+      and return `["a logical message"]`
+
+      if value of the `fieldName` is not string[], 
+      it will generate the first meaningful message and return it as string[]
+  */
+  generateSpecificFieldError(
+    errorInfo: Exception | null | undefined
+  ): (fieldName: string) => string[] | undefined {
+    const errorDetail = getErrorDetail(errorInfo);
+
+    if (errorDetail) {
+      return function getError(fieldName: string) {
```

Comment:
I think it can be done in a utility function. It'll increase readability.

```suggestion
      return function errorGenerator(fieldName: string) {
        return generateErrorFromErrorDetail(errorDetail);
      }
```

Reply:
Also, should we wrap this inside a trycatch block and call `onUnexpectedException` with type `CAUGHT_ERROR_WHEN_GENERATING_SPECIFIC_FIELD_ERROR_MESSAGE`?